spyql/writer.py

```diff
@@ -53,6 +57,17 @@ def flush(self):
         pass
 
 
+class InteractiveWriter(Writer):
```

Comment:
What about using or extending the class `CollectWriter`?
https://github.com/dcmoura/spyql/blob/c379afa89c883b29bc4d3eaf3eea5a7c9d9cbd91/spyql/writer.py#L94

Reply:
Yeah, I missed this. I am removing the Interactive writer and using CollectWriter.